test/extended/cli/project.go

```diff
@@ -21,4 +27,111 @@ var _ = g.Describe("[sig-cli] oc project", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(out).To(o.ContainSubstring("foo=bar"))
 	})
+
+	g.It("can switch between different projects [apigroup:authorization.openshift.io][apigroup:user.openshift.io]", func() {
+		g.By("check auth usage is correct")
+		_, err := oc.Run("policy", "who-can").Args("get", "pods", "-n", "missing-ns").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		_, err = oc.Run("auth", "can-i").Args("get", "pods", "-n", "missing-ns").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		_, err = oc.Run("auth", "can-i").Args("--list", "-n", "missing-ns").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		testProjectUser := "bob-" + oc.Namespace()
+
+		_, err = oc.Run("create", "user").Args(testProjectUser).Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		defer func() {
+			err = oc.Run("delete", "user").Args(testProjectUser).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+		}()
+
+		out, err := oc.Run("auth", "can-i").Args("get", "pods", "--as="+testProjectUser, "-n", "missing-ns").Output()
+		o.Expect(err).To(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("no"))
+
+		_, err = oc.Run("auth", "can-i").Args("--list", "--as="+testProjectUser, "-n", "missing-ns").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By("projects with invalid argument should fail")
+		out, err = oc.Run("projects").Args("test_arg").Output()
```

Comment:
Based on https://github.com/openshift/oc/blob/master/pkg/cli/projects/projects.go running `oc projects` queries projects. Meaning `[apigroup:project.openshift.io]` is used as well.

Reply:
Thanks. Done.